apps/android/app/src/main/java/chat/simplex/app/SimplexService.kt

```diff
@@ -215,7 +215,7 @@ class SimplexService: Service() {
     suspend fun stop(context: Context) = serviceAction(context, Action.STOP)
 
     private suspend fun serviceAction(context: Context, action: Action) {
-      if (!AppPreferences(context).runServiceInBackground.get()) { return }
+      if (!AppPreferences(context).runServiceInBackground.get() || AppPreferences(context).chatWasStopped.get()) { return }
```

Comment:
why is it a preference? I thought it's m.chatRunning? So if it's stopped it would persist after restart? I don't think it should.

Also, why is it needed at all? You could just disable runServiceInBackground preference.